tacacs_plus/__init__.py

```diff
@@ -492,14 +661,15 @@ def send(self, body, seq_no=1):
             )
         raise socket.timeout
 
-    def authenticate(self, username, password,
+    def authenticate(self, username, password, priv_lvl=TAC_PLUS_PRIV_LVL_MIN,
```

Comment:
My idea was to support the case, when this parameter is passed to authenticate method and used in  authentication start packet.